monorepo-migration/generate-replace-message-expressions.py

```diff
@@ -36,7 +36,7 @@ def main():
         repo_numbers = json.load(f)
 
     regexes = [
-        "regex:^==>[focus] ",
+        "regex:^==>[fenix] ",
```

Comment:
ℹ️ Let's run `sync-fenix.sh` once before merging this PR. It will regenerate `monorepo-migration/data/message-expressions.txt` with the new regexes in the right order. This is what the generated commit should look like: https://github.com/mozilla-mobile/firefox-android/commit/a6bd9bd2c9520a08f30826535915e61ffb62a157